source/Mlos.Model.Services/Spaces/Hypergrids.cs

```diff
@@ -20,7 +20,16 @@ namespace Mlos.Model.Services.Spaces
     /// </summary>
     public abstract class IHypergrid
     {
-        public abstract string ToJson();
+    }
+
+    /// <summary>
+    /// Structure describing subgrid joins.
+    /// </summary>
+    public struct SubgridJoin

```

Comment:
Throughout Python code this is called a "JoinedSubgrid" and the two fields are "subgrid" and "join_dimension", why change it here?

Reply:
Not my decision. This is how it was named elsewhere in the C# code. I just moved it and kept the name.

Reply:
This is how the C# was when I arrived. I simply moved SubgridJoin from an internal to a public struct. 

I was also confused by the name SubgridJoin.

Reply:
You have the power to change things for the better :)

Reply:
[zaporter/grpc de5d9879] SubgridJoin => JoinedSubgrid, Deleted most references to JSON serialization